### DISCUSSION TITLE
Welcome to midiremote-userscripts Discussions!

### DISCUSSION THREAD

**Discussion by jtmidi** [steinbergmedia/midiremote-userscripts#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by misohoza** [comment]
Hi @jtmidi 

Is anyone managing the pull requests here?

**Reply by jtmidi** [reply]
Yes, that'll be me ;-)
And to be honest, I rarely find the time to manage the github repo as I am mostly focused on Cubase 12 maintenance.
And as I cannot judge the scripts when I don't have the hardware, we should find a way to have more contributors with rights for PR-merges.
